### PR TITLE
l10n: instructions to maintain the Weblate glossary

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Instructions to maintain the Weblate glossary

## Testing

* make docs-lint

## Deployment

Development documentation only.

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/development/l10n.rst [hunk 1]
```diff
@@ -293,6 +293,32 @@ like Google, Paypal and Twitter now offer two-factor authentication.
...
+For each string to be translated, Weblate shows a glossary of terms
+and their translation to help unify their translations. For instance
+when translating `Please wait for a new two-factor token before
+logging in again`, Weblate notices the word `two-factor` is found in
+the glossary and displays the translation in the `glossary` to the
+right.
+
+|Weblate glossary show page screenshot|
+
+Before translating strings, it is recommended to add all term in the
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: `all term` -> `all terms`

File: docs/development/l10n.rst [hunk 2]
```diff
@@ -293,6 +293,32 @@ like Google, Paypal and Twitter now offer two-factor authentication.
...
+`SecureDrop localization glossary <#glossary>`_ by clicking on the pen
+in the right corner of the glossary displayed with each translated
+string and then `Add new word`:
+
+|Weblate glossary add page screenshot|
+
+When done review and fix the glossary.
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: What do you mean by fix the glossary here?

Comment (dachary) [reply docs/development/l10n.rst]: I updated that (too) terse sentence with:

<pre>
When all the terms are in the glossary, it is recommended to take
another look at the full list of terms and verify there is no
duplicate or other mistakes.
</pre>

File: docs/development/l10n.rst (hunk 1 above)
Comment (dachary) [reply docs/development/l10n.rst]: done. Funny that was the same mistake in another PR. The **s** does not suit **term** apparently ;-)

**Comment by dachary** [comment]
repushed @redshiftzero thanks for the quick review !

**Comment by redshiftzero** [comment]
Looks like a flake (a timeout) in the staging environment:

```
TASK [build-ossec-deb-pkg : Download OSSEC tarball.] ***************************
fatal: [build]: FAILED! => {"changed": false, "dest": "/tmp/build/ossec-hids-2.8.2.tar.gz", "failed": true, "msg": "Connection failure: ('The read operation timed out',)", "state": "absent", "url": "https://github.com/ossec/ossec-hids/archive/2.8.2.tar.gz"}
	to retry, use: --limit @/root/sd/install_files/ansible-base/securedrop-staging.retry
```

(Note that as of the recent merge of #2149, we can prefix branches with `docs-` in order to skip the  staging tests on Circle CI and just run the documentation linting)

**Comment by dachary** [comment]
> Note that as of the recent merge of #2149, we can prefix branches with docs- in order to skip the staging tests on Circle CI and just run the documentation linting)

Oh, right, thanks for reminding me, completely forgot.

**Comment by redshiftzero** [comment]
Hmm staging is passing now, but now the coveralls check continues to show as pending...

**Comment by redshiftzero** [comment]
OK must have just been an unusually long delay on the `coveralls` side. Merging!

**Comment by dachary** [comment]
\o/ <3 @redshiftzero

**Pull request merged**